src/ReleaseHistory.md

```diff
@@ -193,10 +193,12 @@
 * Provide SARIF v1.0 object model and v1 <-> v2 transformation API
 
 ## **v2.0.0-csd.1.0.1** [Sdk](https://www.nuget.org/packages/Sarif.Sdk/2.0.0-csd.1.0.1) | [Driver](https://www.nuget.org/packages/Sarif.Driver/2.0.0-csd.1.0.1) | [Converters](https://www.nuget.org/packages/Sarif.Converters/2.0.0-csd.1.0.1)) | [Multitool](https://www.nuget.org/packages/Sarif.Multitool/2.0.0-csd.1.0.1))
-* Fix weakly typed CreateNotification calls and make API more strongly typed (BREAKING CHANGE)
-* Rename OptionallyEmittedData.ContextCodeSnippets to ContextRegionSnippets (BREAKING CHANGE)
-* Eliminate result.ruleMessageId (in favor of result.message.messageId) (BREAKING CHANGE)
+* BREAKING CHANGE: Fix weakly typed CreateNotification calls and make API more strongly typed
+* BREAKING CHANGE: Rename OptionallyEmittedData.ContextCodeSnippets to ContextRegionSnippets
+* BREAKING CHANGE: Eliminate result.ruleMessageId (in favor of result.message.messageId)
 
-## **v2.0.0-csd.1.0.2** [Sdk](https://www.nuget.org/packages/Sarif.Sdk/2.0.0-csd.1.0.1) | [Driver](https://www.nuget.org/packages/Sarif.Driver/2.0.0-csd.1.0.1) | [Converters](https://www.nuget.org/packages/Sarif.Converters/2.0.0-csd.1.0.1)) | [Multitool](https://www.nuget.org/packages/Sarif.Multitool/2.0.0-csd.1.0.1))
+## **v2.0.0-csd.1.0.2** [Sdk](https://www.nuget.org/packages/Sarif.Sdk/2.0.0-csd.1.0.2) | [Driver](https://www.nuget.org/packages/Sarif.Driver/2.0.0-csd.1.0.2) | [Converters](https://www.nuget.org/packages/Sarif.Converters/2.0.0-csd.1.0.2)) | [Multitool](https://www.nuget.org/packages/Sarif.Multitool/2.0.0-csd.1.0.2))
 * Bugfix to result matching algorithm where empty or null previous log sets caused a NullReferenceException.
 * Bugfix to result matching algorithm where we were incorrectly detecting duplicate data across files, and changed a "NotImplementedException" to the correct "InvalidOperationException".
+* BREAKING CHANGE: invocation.workDirectory is now a FileLocation object (and not a URI expressed as a string)
+* Add run.externalFiles object to schema
```

Comment:
>externalFiles [](start = 10, length = 13)

We've already published NuGet packages 1.0.2, and we don't yet have `run.externalFiles`. That will have to be in a future version.

Reply:
Got it.

---
In reply to: [214740439](https://github.com/Microsoft/sarif-sdk/pull/1067#discussion_r214740439) [](ancestors = 214740439)